config/commands.js

```diff
@@ -1916,7 +1916,8 @@ var commands = exports.commands = {
 		if (!target) return this.parse('/help dice');
 		if (!this.canBroadcast()) return;
 		var d = target.indexOf("d");
-		if (d >= 0) {
+		if (d > -1) {
```

Comment:
I was no aware of this. I believe I compared to -1 in a few places for movesearch and dexsearch; would you like me to change those to ```>= 0``` as well?